multiplayer/src/epics/setCustomIconAsync.ts

```diff
@@ -0,0 +1,23 @@
+import { IconType } from "../types";
+
+import { dispatch } from "../state";
+import {
+    setPresenceIconOverride,
+    setReactionIconOverride,
+} from "../state/actions";
+
+export async function setCustomIconAsync(
+    iconType: IconType,
+    slot: number,
+    pngDataUri?: string
+) {
+    try {
```

Comment:
Would it make sense to have a telemetry tick here? It might not, if we're doing this automatically, but if we knew whether or not it was done explicitly by the user we could just log those.

Reply:
the implicit / explicit bit isn't passed around right now -- I see it more like an `!important` where most of the time the default is what you want, and it's just for the special case where you want a little more control that you would use another block, so outside the simulator they're treated the same.

That said happy to add a tick here but it would be a bit noisy - a standard, full multiplayer game would usually send 16 ticks per play of the game (i.e. would resend them on restart, as it stems from user code that will typically be in the on start event)

Reply:
Makes sense, thanks.